modules/datadog-synthetics/README.md

```diff
@@ -0,0 +1,185 @@
+# Component: `datadog-synthetics`
+
+This component is responsible for provisioning Datadog synthetic tests.
+
+Synthetic tests allow you to observe how your systems and applications are performing using simulated requests and actions
+from the managed locations around the globe and to monitor internal endpoints from private locations.
+
+## Usage
+
+**Stack Level**: Regional
+
+Here's an example snippet for how to use this component:
+
+### Stack Configuration
+
+```yaml
+components:
+  terraform:
+    datadog-synthetics:
+      metadata:
+        component: "datadog/synthetics"
+      settings:
+        spacelift:
+          workspace_enabled: true
+      vars:
+        enabled: true
+        name: "datadog-synthetics"
+        locations:
+          - "all"
+        synthetics_paths:
+          - "catalog/synthetics/defaults/*.yaml"
+        synthetics_private_location_component_name: "datadog-synthetics-private-location"
+        private_location_test_enabled: true
+```
+
+### Synthetic YAML Configuration (pointed to by the `synthetic_paths` variable)
```

Comment:
Where can the user find more details on how to configure this? E.g. even by reading docs or terraform provider docs.